packages/i18nify-playground/src/sections/phoneNumber/phoneNumber-form.jsx

```diff
@@ -82,15 +84,15 @@ const PhoneNumberForm = ({
                     >
                       <div width="30px">+ {code}</div>
                       <img
-                        width="40px"
+                        width="28px"
                         style={{
                           marginLeft: 8,
                           marginRight: 8,
                         }}
                         loading="lazy"
-                        src={`/assets/flags/${dialCodeCountryCodeMap[
+                        src={getFlagOfCountry(dialCodeCountryCodeMap[
```

Comment:
Test it properly, getting below error while testing:

```
Uncaught i18nify Error: Error: Invalid country code: AC
```

Reply:
if you comment these flags in dialCodeMap and dialCodeCountryCodeMap in phoneNumber.js. No error will be there

Reply:
I have stored the path in constants and using the variable to make the url. This way we won't get an error. Have tested the same. Please check now. @RgnDunes Thanks!